app/styles/modules/_marketing-ios.scss

```diff
@@ -12,9 +12,14 @@
     }
 
     li {
-      display: block;
+      display: inline-block;
       list-style-type: none;
       margin-bottom: 10px;
+      padding: 5px;
+    }
+    
+    span {
```

Comment:
`span`s are among our most common elements and because of the way CSS parsing works, using simply a `span` as a selector will slow down performance. [This article](https://css-tricks.com/efficiently-rendering-css/) is a bit dated but explains how a browser's CSS parser works at a high level, right to left in selectors, and general element (span, li, etc) are the least efficient, id the most.

Can you add a class name to the span, and use that as the selector instead? Maybe `download-firefox-mobile`.